dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts

```diff
@@ -78,25 +68,25 @@ describe("As a user, I expect the search functionality on the /companies page to
   it("Scroll the page and check if search icon and search bar behave as expected", () => {
     cy.visitAndCheckAppMount("/companies");
     cy.get("input[name=search_bar_top]").type("a").type("{enter}");
-    cy.get("button[name=search_bar_collapse]").should("not.exist");
+    cy.get("button[name=search_bar_collapse]").should("not.be.visible");
```

Comment:
why?    aren't they v-ifed?   threfore not exist should also be the case and be a stricter test condition

Reply:
No they are v-show now.